README.md

```diff
@@ -89,6 +90,7 @@ This action is primarily designed for use with GitHub.com's Actions workflows an
 
 | Release | GHES Compatibility |
 |:---|:---|
+| [`v4`](https://github.com/actions/deploy-pages/releases/tag/v4) | :warning: none |
```

Comment:
Maybe call out that is planned?

```suggestion
| [`v4`](https://github.com/actions/deploy-pages/releases/tag/v4) | :information_source: TBD |
```

Reply:
yeah, that's better

Reply:
LOL, my new suggestions aims for something of a combo. 😜